packages/core/test/request-builder/request/get-filters.spec.ts

```diff
@@ -151,4 +158,21 @@ describe('get filters', () => {
         .filter
     ).toBe(testFilterFunctionNested.odataStr);
   });
+
+  describe('odata v4', () => {
+    it('for simple filters', () => {
```

Comment:
repharse the assertion in a more descriptive way `it('should do ABC for simple filters...`

Reply:
Agreed.

Reply:
in general, I agree.
However, in this spec file, all tests are about get filter for different cases like
- simple filter
- string field
- complex field
- filter function
Then, it will make lots of useless in terms of the part of `should do ABC`, since it's quite obvious.
I guess that's the reason, why in this spec file all of the tests are using `it('for some case')`.